### DISCUSSION TITLE
Combos as bind keys

### DISCUSSION THREAD

**Discussion by doesntfazer** [hyprwm/Hyprland#8242] [General]
There's a feature in QMK where if you press multiple keys simultaneously, it outputs whatever you want. It would be nice to implement something similar in Hyprland. 
For example, press Q+W at the same time (or withing a 40ms window) and it opens terminal.
Obviously this is something that could technically be achieved using a QMK keyboard and binding a combo to output super+q but it would be cool to see it just natively work with any keyboard.
Just a thought.

**Comment by fufexan** [comment]
Not in scope. Use [kanata](https://github.com/jtroo/kanata/) or [kmonad](https://github.com/kmonad/kmonad), these programs are like QMK but for non-QMK keyboards.
